packages/material-ui-lab/src/TreeItem/TreeItem.test.js

```diff
@@ -209,34 +209,57 @@ describe('<TreeItem />', () => {
     });
 
     describe('aria-selected', () => {
-      it('should have the attribute `aria-selected=false` if not selected', () => {
-        const { getByTestId } = render(
-          <TreeView>
-            <TreeItem nodeId="test" label="test" data-testid="test" />
-          </TreeView>,
-        );
+      describe('single-select', () => {
+        it('should not have the attribute `aria-selected` if not selected', () => {
```

Comment:
Why should it not have the attribute has opposed to `aria-selected="false"`?

Reply:
I am going from https://www.w3.org/TR/wai-aria-practices/#TreeView

> If the tree does not support multiple selection, aria-selected is set to true for the selected node and it is not present on any other node in the tree.

Reply:
Alright. Please provide some context in the future if you already knew this. Reduces review friction.

Reply:
Alright. Please provide some context in the future if you already knew this. Reduces review friction.

Reply:
It is mentioned in the associated issue #20102 ( although that really is two issues )

Reply:
Sure but when I revisit this later I don't want to browse through the conversation in a linked PR. The reason for a change should be immediate.

Reply:
Should I have put the issue number in the test description ?